### PR TITLE
Monitor queued jobs

### PR DESCRIPTION
**Pull request by AsemAlalami**
Monitor queued jobs https://github.com/romanzipp/Laravel-Queue-Monitor/discussions/62

also, I think it fixed the #76 issue

the PR adds "Initial Data" feature that initializes custom data to the monitor job

### CODE REVIEW

**Comment by romanzipp** [comment]
Nice work! Thanks!
What do you think about adding a config option which needs to be implicitly enabled to enable monitoring queued jobs?

**Comment by AsemAlalami** [comment]
Yes, I can do that 
But wonder why someone wouldn't want to monitor queued jobs?!

File: src/Services/QueueMonitor.php [hunk 1]
```diff
@@ -174,6 +218,17 @@ protected static function jobFinished(JobContract $job, int $status, ?\Throwable
             return;
         }
 
+        // if the job has an exception, but it's not failed (it did not exceed max tries and max exceptions),
```
Comment (AsemAlalami) [inline src/Services/QueueMonitor.php:221]: @romanzipp please take a look here
should we consider these cases as queued when the config option is disabled?

actually, Laravel returns these cases to the queue but I mean when the config option is disabled

**Comment by AsemAlalami** [comment]
I added a config option, but please take a look to my comment

**Comment by romanzipp** [comment]
Do you have anything to add or is this ready to merge=

**Comment by AsemAlalami** [comment]
I think it's ready to merge

**Comment by romanzipp** [comment]
Just a FYI, there are tests failing with the upcoming version. I'm looking into it but if you have an idea feel free to file a PR!

https://github.com/romanzipp/Laravel-Queue-Monitor/actions/runs/6071786840/job/16470512909

**Pull request merged**